spec/services/spree_klaviyo/create_or_update_profile_spec.rb

```diff
@@ -3,6 +3,10 @@
 RSpec.describe SpreeKlaviyo::CreateOrUpdateProfile do
   subject { described_class.call(klaviyo_integration: klaviyo_integration, user: user) }
 
+  before do
```

Comment:
@jawaria-sparksol please create user with `accepts_email_marketing` true instead of this